### DISCUSSION TITLE
MQTT Baby Crying Message

### DISCUSSION THREAD

**Discussion by randyesperben** [roleoroleo/yi-hack-Allwinner-v2#555] [Q&A]
Hi all,

I have firmware base 12.0.35.8_202211241029, using y291ga version of Allwinner, and everything seems to working well but Im testing MQTT and I am unable to receive baby crying message on mqtt service.  Any ideas or suggestions?

Many thanks in advance!

**Comment by roleoroleo** [comment]
I think that baby crying is no longer available with fw 12.
Do you have the option in the official app?

**Comment by randyesperben** [comment]
Well, there is an option "loud noises", and gives as examples baby crying. But no, is not a unique option for babies.

**Reply by roleoroleo** [reply]
I think they removed it.
